using-mobile-extensions/mobile-core/identity/identity-api-reference.md

```diff
@@ -520,6 +522,8 @@ If an error occurs while retrieving the URL string, _callback_ will be called wi
   * `TS` - A timestamp taken when this request was made
 * The optional `adobe_aa_vid` attribute is the URL-encoded Analytics Custom Visitor ID \(VID\), if previously set in the [Analytics extension](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-analytics).
 
+When `AdobeCallbackWithError` is provided, a 500ms timeout is used when fetching the attributes described above from the Mobile SDK. If the operation times out or an unexpected error occurs, the `fail` method is called with the appropriate `AdobeError`.
```

Comment:
Please reword as follows:
"When AdobeCallbackWithError is provided, and you are fetching the attributes from the Mobile SDK, the timeout value is 500ms. If the operation times out or an unexpected error occurs, the fail method is called with the appropriate AdobeError.